src/nrniv/multisplit.cpp

```diff
@@ -393,8 +391,10 @@ void MultiSplitControl::multisplit(Section* sec, double x, int sid, int backbone
 	assert(root);
 //	printf("is %s\n", secname(root->sec));
 	MultiSplit* ms;
-	if (classical_root_to_multisplit_->find(root, ms)) {
-		if (backbone_style == 2) {
+	const auto& msiter = classical_root_to_multisplit_->find(root);
```

Comment:
This is correct only if classical_root_to_multisplit is not NULL. That check was done by the previous macro version. In this case it is never NULL because of line 381. Sadly it I can require some cognitive effort to decide in each case if NULL is a reasonable value which should skip the body of the for or if.